snap/snapcraft.yaml

```diff
@@ -150,12 +142,11 @@ apps:
     plugs: [network]
   # This is a one-shot service which sets up consul's ACL and prepare for creating consul's agent tokens later on
   security-consul-bootstrapper:
-    adapter: none
     after:
       - security-secretstore-setup
     command: bin/setup-consul-acl.sh
     command-chain:
-      - bin/service-config-overrides.sh
+      - bin/source-env-file.sh
     environment:
       ENV_FILE: $SNAP_DATA/config/security-bootstrapper/res/security-bootstrapper.env
```

Comment:
`ENV_FILE` should be changed to `$SNAP_DATA/config/security-bootstrapper/overrides.env`